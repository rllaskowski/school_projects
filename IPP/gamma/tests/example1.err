ERROR 2
ERROR 3
ERROR 4
ERROR 5
ERROR 6
ERROR 7
ERROR 8
ERROR 9
ERROR 10
ERROR 11
ERROR 14
ERROR 15
ERROR 16
ERROR 17
ERROR 18
ERROR 19
ERROR 20
ERROR 21
ERROR 22
ERROR 23
ERROR 24
ERROR 25
ERROR 26
ERROR 27
ERROR 28
ERROR 29
ERROR 30
ERROR 31
ERROR 32
ERROR 33
ERROR 34
ERROR 35
ERROR 36
ERROR 37
ERROR 38
ERROR 39
ERROR 40
ERROR 41
ERROR 42
ERROR 43
ERROR 44
ERROR 45
ERROR 48
ERROR 49
ERROR 50
ERROR 51
ERROR 53
ERROR 54
ERROR 55
ERROR 56
ERROR 58
ERROR 59
ERROR 60
ERROR 61
ERROR 63
ERROR 64
ERROR 65
ERROR 66
ERROR 68
ERROR 69
ERROR 72
ERROR 73
ERROR 74
ERROR 75
ERROR 76
ERROR 77
ERROR 78
ERROR 79
ERROR 82
ERROR 83
ERROR 84
ERROR 85
ERROR 86
ERROR 87
ERROR 88
ERROR 102
ERROR 103
ERROR 104
ERROR 105
ERROR 106
ERROR 107
ERROR 110
ERROR 111
