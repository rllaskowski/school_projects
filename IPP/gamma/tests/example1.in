# zła liczba parametrów
I
B
I 1
B 1
I 1 1
B 1 1
I 1 1 1
B 1 1 1
I 1 1 1 1 1
B 1 1 1 1 1

# parametr o złej wartości
I -1 1 1 1
B -1 1 1 1
I 1 -1 1 1
B 1 -1 1 1
I 1 1 -1 1
B 1 1 -1 1
I 1 1 1 -1
B 1 1 1 -1
I -2147483640 1 1 1
B -2147483640 1 1 1
I 1 -2147483640 1 1
B 1 -2147483640 1 1
I 1 1 -2147483640 1
B 1 1 -2147483640 1
I 1 1 1 -2147483640
B 1 1 1 -2147483640
I 4294967296 1 1 1
B 4294967296 1 1 1
I 1 4294967296 1 1
B 1 4294967296 1 1
I 1 1 4294967296 1
B 1 1 4294967296 1
I 1 1 1 4294967296
B 1 1 1 4294967296
I 9999999999 1 1 1
B 9999999999 1 1 1
I 1 9999999999 1 1
B 1 9999999999 1 1
I 1 1 9999999999 1
B 1 1 9999999999 1
I 1 1 1 9999999999
B 1 1 1 9999999999

# parametr niebędący liczbą
I a 1 1 1
B b 1 1 1
I 1c 1 1 1
B 1d 1 1 1

I 1 A 1 1
B 1 B 1 1
I 1 1C 1 1
B 1 1D 1 1

I 1 1 < 1
B 1 1 > 1
I 1 1 1= 1
B 1 1 1? 1

I 1 1 1 p
B 1 1 1 q
I 1 1 1 1r
B 1 1 1 1s

I 1 1 1 1 x
B 1 1 1 1 y

# parametr nieakceptowalny przez gamma_new
I 0 2 2 2
B 0 2 2 2
I 2 0 2 2
B 2 0 2 2
I 2 2 0 2
B 2 2 0 2
I 2 2 2 0
B 2 2 2 0

# niepoprawne polecenie przed wejściem w tryb wsadowy lub interaktywny
m 1 1 1
g 1 1 1
b 1
f 1
q 1
p
x

# przejście do trybu wsadowego
B 4 4 2 3

# jakaś prosta gra
m 1 2 2
g 1 3 3
b 1
f 1
q 1
p

# kilka błędów
m 1
g 1
b
f
q
p 1

# zmiana trybu niemożliwa
I 4 4 2 3
B 4 4 2 3
