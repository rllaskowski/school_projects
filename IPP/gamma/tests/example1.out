OK 91
1
0
1
15
0
....
..1.
....
....
